DQMOffline/Configuration/python/DQMOffline_cff.py

```diff
@@ -46,11 +46,10 @@
 from DQM.Physics.DQMTopMiniAOD_cff import *
 from Validation.RecoTau.DQMSequences_cfi import *
 from DQM.TrackingMonitorSource.TrackingSourceConfig_Tier0_cff import *
-from DQM.TrackingMonitorSource.pixelTracksMonitoring_cff import *
```

Comment:
another revert of code from #22395

it looks like a rebase was made incorrectly

Reply:
I will fix this. Thanks for noticing it